### DISCUSSION TITLE
Help with passiing a hip report

### DISCUSSION THREAD

**Discussion by kkempter-getswift** [yuezk/GlobalProtect-openconnect#389] [General]
All;

Can someone pass me an example of how to pass a hip report to the gpclient cli?

Thanks in advance

**Comment by yuezk** [comment]
@kkempter-getswift Try `sudo gpclient connect <portal> --hip` or `sudo gpclient connect <portal> --csd-wrapper path/to/your/hip/script` if you are using your customized hip report script.
